styles/calendar_styles.css

```diff
@@ -6,10 +6,9 @@ body {
 .container {
     display: grid;
     grid-template-columns: 4.5vw 91vw 4.5vw;
-    grid-template-rows: 5vh 5vh 80vh 5vh 55vh;
+    grid-template-rows: 5vh 85vh 5vh 5vh;
```

Comment:
I'm not sure we should be using `vh`/`vw `units for this, but it probably works so it's probably fine.